style/Plain/ui_800x480.json

```diff
@@ -397,44 +397,45 @@
                         "id": "graph",
                         "type" : "ClutterActor",
                         "width": 800,
-                        "height": 400,
-                        "y": 40
+                        "height": 350,
+                        "y": 60,
+                        "x": 10
                     },{
                         "id": "heat-tool0",
                         "type": "MxButton",
                         "style-class": "heat",
                         "is_toggle": true,
                         "x": 284,
                         "y": 400,
-                        "width": 64,
-                        "height": 64
+                        "width": 50,
+                        "height": 50
                     },{
                         "id": "heat-bed",
                         "type": "MxButton",
                         "style-class": "heat",
                         "is_toggle": true,
                         "x": 368,
                         "y": 400,
-                        "width": 64,
-                        "height": 64
+                        "width": 50,
+                        "height": 50
                     },{
                         "id": "heat-tool1",
                         "type": "MxButton",
                         "style-class": "heat",
                         "is_toggle": true,
                         "x": 452,
                         "y": 400,
-                        "width": 64,
-                        "height": 64
+                        "width": 1,
```

Comment:
Is this to hide the button? If so, I think it would be better to do it programatically based on the number of extruders.

Reply:
I agree. Otherwise you keep the one pixel that may trigger an unwanted event without realizing it.

Reply:
Yes it was to hide it sure is better to hide it in extruder number

Reply:
Could you change this to the same size? I know it's annoying if you only have one extruder, but it can't be accepted as it is.

Reply:
ok width set to 50 same as others